ui/shared/AddressInput.qml

```diff
@@ -0,0 +1,89 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import QtGraphicalEffects 1.13
+import "../imports"
+
+Item {
+    id: root
+    property string validationError: "Error"
+    property alias label: inpAddress.label
+    property string selectedAddress
+
+    height: inpAddress.height
+
+    function isValidAddress(inputValue) {
+        return /0x[a-fA-F0-9]{40}/.test(inputValue)
+    }
+    function isValidEns(inputValue) {
```

Comment:
I'd put those two functions in `Utils.qml`

Reply:
`isValidEns` needs to change at some point, and it will not go in `Utils`, but I have moved the address regex to Utils.

Reply:
Also, note I updated the regex in `Utils.isHex` to include uppercase characters as well. Please let me know if this was an incorrect assumption.

Reply:
Sounds good